website/siteConfig.js

```diff
@@ -3,48 +3,11 @@
 
 const repoUrl = 'https://github.com/disneystreaming/weaver-test';
 
-const siteConfig = {
-  title: 'Weaver Test',
-  tagline: 'A lean test-framework built on top of cats-effect and fs2',
-  url: 'http(s)://disneystreaming.github.io/',
-  baseUrl: '/weaver-test/',
-  projectName: 'weaver-test',
-  organizationName: 'disneystreaming',
-  githubHost: 'github.com',
-  repoUrl: repoUrl,
-  // For no header links in the top nav bar -> headerLinks: [],
-  separateCss: ["api"],
-  headerLinks: [
-    {doc: 'installation', label: 'Docs'},
-    { href: repoUrl, label: "GitHub", external: true },
-  ],
-
-  headerIcon: 'img/dss-profile-white-transparent.svg',
-  favicon: 'img/dss-profile-white-transparent.svg',
+const siteConfigJson = require("./siteConfig.json");
```

Comment:
extracted all static bits from the siteConfig into a json file to be able to read it from scala